src/main/groovy/com/netgrif/application/engine/petrinet/domain/dataset/logic/action/ActionDelegate.groovy

```diff
@@ -787,6 +812,17 @@ class ActionDelegate {
         change useCase.getField(fileFieldId) value { new FileFieldValue(filename, storagePath) }
     }
 
+    void generatePDF(String transitionId, FileField fileField) {
```

Comment:
how does the introduction of these methods relate to the changes made to these methods in PRs #39 and #40 ?

Reply:
@renczesstefan informed me that he will resolve conflicted changes made to these methods once the related PRs are ready to merge